app/ChatPanelHeader.tsx

```diff
@@ -124,6 +167,77 @@ function ChatPanelHeader() {
           {currentThreadName}
         </Button>
       )}
+
+      {/* Insights dropdown */}
+      {widgetAnchors.length === 0 ? (
+        <Tooltip content="Ask a question to generate insights" showArrow>
+          <span style={{ display: "inline-flex" }}>
+            <Button variant="ghost" size="sm" disabled>
+              Go to insight
+              <CaretDownIcon />
+            </Button>
+          </span>
+        </Tooltip>
+      ) : (
+        <Menu.Root>
+          <Menu.Trigger asChild>
+            <Button variant="ghost" size="sm">
```

Comment:
```suggestion
            <Button
              variant="ghost"
              bgGradient="to-br"
              gradientFrom="blue.300/25"
              gradientTo="lime.300/25"
              size="sm"
            >
```
Adds button gradient